packages/neuron-wallet/src/channel/index.ts

```diff
@@ -256,12 +256,22 @@ export class Listeners {
     return ipcMain.on(
       Channel.CheckWalletPassword,
       (e: Electron.Event, { walletID, password }: { walletID: string; password: string }) => {
+        let target = -1
         let valid = false
-        wallets().forEach((value: Wallet) => {
-          if (value.id === walletID && value.password === password) {
-            valid = true
+        wallets().forEach((value: Wallet, index: number) => {
+          if (value.id === walletID) {
+            target = index
           }
         })
+        if (target === -1) {
+          const notification = new Notification({
+            title: `Check Wallet Password`,
+            body: `Wallet not find`,
+          })
+          notification.show()
+        } else if (wallets()[target].password === password) {
```

Comment:
variable `valid` is redundant, just put `send success message` in this logic branch